### ISSUE TITLE
LUA Reverting After Insert of Config Settings; MP Hosted

### ISSUE THREAD

**Issue by diridio** [TEHE-Studios/ExpandedHelicopterEvents#266]
**Describe the bug**

I followed the instructions of the FAQ in inserting the config settings directly into my hosted server's .lua, shown in the screenshots is exactly what my ,lua looks like prior to launching the server. Once I and my friends join, we don't get a single helicopter event. Checked the .lua again, and I find out that all the config files I added in past FuelAPI is deleted. Furthermore, it resets my .lua to default settings, which turns off things that I had to turn on myself, such as multi-hit and mini-maps.

**Screenshots**

![image](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/assets/152043390/ae888035-5671-435b-8c76-b555f79d9a48)

**Console logs:**

[console - Copy.txt](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/files/13494844/console.-.Copy.txt)

**Comment by 1Danth** [comment]
I have the same problem on my hosted server. I tried to connect to the .lua using directly FTP but don't work too. I tried to set a config that increase frequencies to see whether spawn helicopters or at least if the options are working, but nothing happened. I didn't try to set permissions "only read" on .lua file to see if it forces it to set or something because, I am worried if changing permissions affects in game.
